scripts/update-terraform-modules.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+## Init temp dir
+tmp_dir=tmp
+mkdir -p $tmp_dir
+
+github_org=cloudposse
+github_api_limit=100
+github_api_pages=3
+
+modules=[]
+
+## Fetch repository names
+echo "Fetching repository names..."
+if [ ! -f $tmp_dir/modules_cache.txt ]; then
+    modules=$(
+        seq $github_api_pages | \
+        xargs -I "{}" curl --fail -sSL "https://api.github.com/orgs/${github_org}/repos?per_page=${github_api_limit}&page={}" | \
+        jq -r '.[] | .name' | \
+        sort | grep terraform-*
+    )
+    echo "$modules" > $tmp_dir/modules_cache.txt
+else
+    modules=$(<$tmp_dir/modules_cache.txt)
+fi
+
+## Fetch terraform-modules content
+echo "Fetching terraform-modules content..."
+for module in $modules
+do
+    module_dir=$tmp_dir/projects/terraform/$module
+    if [ ! -d $module_dir ]; then
+        mkdir -p $module_dir
+        echo ""
+        echo "Fething files for $module..."
+        curl --fail -sSL --write-out "%{url_effective}\n" "https://raw.githubusercontent.com/${github_org}/${module}/master/README.yaml" -o $module_dir/README.yaml
+        mkdir -p $module_dir/docs
+        curl --fail -sSL --write-out "%{url_effective}\n" "https://raw.githubusercontent.com/${github_org}/${module}/master/docs/targets.md" -o $module_dir/docs/targets.md
+        curl --fail -sSL --write-out "%{url_effective}\n" "https://raw.githubusercontent.com/${github_org}/${module}/master/docs/terraform.md" -o $module_dir/docs/terraform.md
```

Comment:
We should "clean up" somewhere as we don't need to preserve the `README.yaml` and `docs/*`

Reply:
done